app/config/easy_admin.yml

```diff
@@ -1,27 +1,34 @@
 easy_admin:
     site_name: 'Chloris'
     design:
-        menu:
-        - User
-        - { label: 'Go to homepage', route: 'homepage' }
-        color_scheme: 'light'
+      color_scheme: 'light'
+      menu:
+          - User
+          - { label: 'Go to homepage', route: 'homepage' }
     entities:
         User:
             class: AppBundle\Entity\User
-            controller: AppBundle\Controller\Admin\UserController
+            controller: EasyAdminBundle\Controller\UserControllerPreventingFromSelfDelete
             list:
                 fields:
                     - property: 'username'
                     - { property: 'email', type: 'email', label: 'E-mail address' }
                     - 'enabled'
                     - 'lastLogin'
             edit:
-                fields: [property: 'username', property: 'email', { property: 'roles', type: 'choice', type_options: { multiple: true, choices: { 'User': 'ROLE_USER', 'Administrator': 'ROLE_ADMIN' } } }]
+                fields:
+                    - { property: 'username' }
+                    - { property: 'email' }
+                    - { property: 'roles', type: 'choice',
+                        type_options: { multiple: true,
```

Comment:
It looks like there is missing closing brace, but it is not. It's just intendation written wrong way.